### ISSUE TITLE
Initial Particle shape

### ISSUE THREAD

**Issue by KatoWitEC** [VincentGarreau/particles.js#278]
Hi Vincent/All particles.js Users,

I would like to intialize particles.js with defined shape. 
As a shape I mean the 'picture' particles are creating by themselves.
Something like here:
https://image.shutterstock.com/z/stock-vector-abstract-brazil-map-lines-connection-vector-illustration-443007856.jpg

After initialization this intial shape should not change, only particles will move some slight distance from their original positions.

Is that doable using this library? Where to start?
Thanks in advance for any advice.

**Comment by ghost** [comment]
Yeah, I would be interested in something similar. Any luck with this or any other library?
Thanks
